requirements.txt

```diff
@@ -2,3 +2,5 @@ ops
 pydantic
 # crossplane is a package from nginxinc to interact with the Nginx config
 crossplane
+
+minio
```

Comment:
Agreed. Would this work with other S3 providers?

Reply:
Removed, but on a side note mimir expects an s3 bucket to exist and I'm wondering which of the below make sense

1. Leave it as the responsibility of the user to actually create a bucket before hand and also to ensure that the s3 connection provided is valid
2. Make use of something like [boto3](https://pypi.org/project/boto3/) to validate the s3 connection and create the bucket if it doesn't exists, but this will make boto3 a dependency tho

Reply:
Will go with option 1 and we can have a separate issue for s3 bucket creation and connection validation if needed